src/aap_eda/core/tasking/__init__.py

```diff
@@ -47,17 +49,35 @@
 ]
 
 
+def get_redis_prefix():
+    for _ in range(50):
+        try:
+            return f"EDA-{service_id()[0:8]}-rq"
+        except ProgrammingError:
+            # service_id() will fail at the first migration when
+            # dab_resource_registry_serviceid has not been created
+            if (
+                len(sys.argv) == 2
+                and sys.argv[0].endswith("aap-eda-manage")
+                and sys.argv[1] in ["migrate", "makeimgrations"]
```

Comment:
`aap-eda-manage` could not be called directly, so `sys.argv` could be more than 2 and `aap-eda-manage` could not be sys.argv[0]`
What about `sys.argv[-1] in ["migrate", "makemigrations"]`

Reply:
Second thought, I think the most reliable way could be
```
any(arg in sys.argv for arg in ["migrate", "makemigrations"])
```